File: Applications/SinglePageApp/src/app/models/inbody/muscle-fat-analysis.ts
import { TestedResult } from './tested-result';

export class MuscleFatAnalysis {
  weight: TestedResult = new TestedResult();
  skeletalMuscleMass: TestedResult = new TestedResult();
  waistHipRatio: TestedResult = new TestedResult();
  visceralFatLevel: TestedResult = new TestedResult();
  bodyFatMass: TestedResult = new TestedResult();
}

File: Domains/ApplicationDomain/Gym/Model/MuscleFatAnalysis.cs
using ApplicationDomain.Common;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.Gym.Model.MyInBody
{
    public class MuscleFatAnalysis
    {
        public TestedResult Weight { set; get; }
        public TestedResult SkeletalMuscleMass { set; get; }
        public TestedResult WaistHipRatio { set; get; }
        public TestedResult VisceralFatLevel { set; get; }
        public TestedResult BodyFatMass { set; get; }

        public MuscleFatAnalysis()
        {
            this.Weight = new TestedResult();
            this.SkeletalMuscleMass = new TestedResult();
            this.WaistHipRatio = new TestedResult();
            this.VisceralFatLevel = new TestedResult();
            this.BodyFatMass = new TestedResult();
        }
    }
}
